stable/mongodb/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: mongodb
-version: 1.0.2
```

Comment:
As you are making backward incompatible changes you should update the major version

Reply:
👍